classes/class3/cat-app/views/cats.handlebars

```diff
@@ -0,0 +1,7 @@
+<h1>Cats Looking for Homes</h1>
+
+<ul>
+{{#each cats}}
+  <li>{{this.name}}</li>
```

Comment:
Why only show the name? Seeing the cats' color in this list as well could help by hinting at which colors would be good to search for when using `/bycolor/`.